boards/hail/src/main.rs

```diff
@@ -39,7 +39,7 @@ static mut SPI_WRITE_BUF: [u8; 64] = [0; 64];
 const NUM_PROCS: usize = 20;
 
 // How should the kernel respond when a process faults.
-const FAULT_RESPONSE: kernel::process::FaultResponse = kernel::process::FaultResponse::Panic;
+const FAULT_RESPONSE: kernel::process::FaultResponse = kernel::process::FaultResponse::Restart;
```

Comment:
Given that a lot of apps (including the "Hail" app) don't restart successfully, I'm not sure we're ready for restart to be the default on Hail.

Reply:
Oops! Didn't mean to include that.